runelite-client/src/main/java/net/runelite/client/plugins/grandexchange/GrandExchangePlugin.java

```diff
@@ -440,21 +452,60 @@ public void onGameTick(GameTick event)
 		final String geTextString = geText.getText();
 		final int itemId = grandExchangeItem.getItemId();
 
+		if (!geText.getText().contains("<br>"))
+		{
+			geText.setText(geText.getText() + "<br>");
+			itemDescription = geText.getText();
+		}
+
 		if (itemId == OFFER_DEFAULT_ITEM_ID || itemId == -1)
 		{
 			// This item is invalid/nothing has been searched for
 			return;
 		}
 
+		if (config.enableAfford())
+		{
+			int currentItemPrice = client.getVar(Varbits.GRAND_EXCHANGE_PRICE_PER_ITEM);
+			if (!geTextString.contains(AFFORD_GE_TEXT))
```

Comment:
I don't understand this whole block. Does this really need to be set every gametick?

Reply:
It will only set the Afford string if:
1. it's not already set (!geTextString.contains(AFFORD_GE_TEXT))
2. Price per item has changed (oldPrice != currentItemPrice)  [update Afford quantity]